### PR TITLE
fix* cigar case verb

### PR DESCRIPTION
**Pull request by Kegdo**
## About The Pull Request

*delete
add description about alt+clicking
## Why It's Good For The Game

the same issue apparently is in the hardcase code, and it seems there is no real fix for this. So. Kill!

## Changelog
:cl:
fix: cigar case open/close verb
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -286,7 +286,7 @@
 
 /obj/item/storage/fancy/cigar
 	name = "cigar case"
-	desc = "A case for holding your cigars when you are not smoking them. Fancy!"
+	desc = "A case for holding your cigars when you are not smoking them. Fancy! Alt+click to open and close."
```
Comment (TheShown911) [inline code/game/objects/items/weapons/storage/fancy.dm:289]: Bruh, there are no things like these in the description, do not put it here.

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 2]
```diff
@@ -307,19 +307,6 @@
 		return 2
 	return 0
 
-
-/obj/item/storage/fancy/cigar/verb/quick_open_close(mob/user)
-	set name = "Close cigar case"
-	set category = "Object"
-	set src in view(1)
```
Comment (TheShown911) [inline code/game/objects/items/weapons/storage/fancy.dm:314]: Would `view(0)` work here?

**Comment by Kegdo** [comment]
Nuh-uh, tried it.

File: code/game/objects/items/weapons/storage/fancy.dm (hunk 1 above)
Comment (Kegdo) [reply code/game/objects/items/weapons/storage/fancy.dm:289]: How to tell about alt+clicking then? Hardcases have it in their desc.

File: code/game/objects/items/weapons/storage/fancy.dm (hunk 2 above)
Comment (Kegdo) [reply code/game/objects/items/weapons/storage/fancy.dm:314]: Problem is, it thinks that every mob is mob/user. No idea why. Same issue in hardcases code.

Comment (hyperioo) [reply code/game/objects/items/weapons/storage/fancy.dm:314]: What do you mean every mob is mob/user? 🤔

Comment (Kegdo) [reply code/game/objects/items/weapons/storage/fancy.dm:314]: image
I mean THIS.
image

**Comment by drexample** [comment]
bruh

**Review by drexample — CHANGES_REQUESTED** [review]
Comply to maintainer requests please

**Comment by Kegdo** [comment]
here?

**Pull request closed without merging**